common/src/lib.rs

```diff
@@ -146,29 +151,24 @@ macro_rules! make_handle {
     };
 }
 
-impl Into<Isometry3<f32>> for Transform {
-    fn into(self) -> Isometry3<f32> {
-        Isometry3 {
-            rotation: self.orient,
-            translation: Translation3::from(self.pos),
-        }
+impl Into<Mat4> for Transform {
+    fn into(self) -> Mat4 {
+        Mat4::from_quat(self.orient) * Mat4::from_translation(self.pos)
     }
 }
 
-impl From<Isometry3<f32>> for Transform {
-    fn from(value: Isometry3<f32>) -> Self {
+impl Mul for Transform {
+    type Output = Self;
+    fn mul(self, rhs: Self) -> Self::Output {
         Self {
-            pos: value.translation.vector.into(),
-            orient: value.rotation,
+            pos: self.pos + self.orient * rhs.pos,
+            orient: self.orient * rhs.orient,
         }
     }
 }
 
-impl Mul for Transform {
-    type Output = Self;
-    fn mul(self, rhs: Self) -> Self::Output {
-        let lhs: Isometry3<f32> = self.into();
-        let rhs: Isometry3<f32> = rhs.into();
-        (lhs * rhs).into()
+impl Default for GenericHandle {
+    fn default() -> Self {
+        Self(0xBAD_BAD_BAD_BAD_BAD_BAD_BAD_BAD_BAD_BAD)
```

Comment:
Bad bad

Reply:
Also, a user won't have to do this right?

Reply:
Nah they don't need to, this is just to make sure that if the default GenericHandle is found in e.g. a debugger it's recognizable